Modules/@babylonjs/react-native-iosandroid/ios/CMakeLists.txt

```diff
@@ -27,6 +27,10 @@ target_include_directories(reactnative INTERFACE ${REACTNATIVE_DIR}/ReactCommon/
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/")
 
 set(BABYLON_NATIVE_BUILD_APPS OFF CACHE BOOL "")
+if (${BASEKIT_BUILD})
+    set(BABYLON_NATIVE_PLUGIN_NATIVEXR OFF CACHE BOOL "")
+    set(BABYLON_NATIVE_PLUGIN_NATIVECAMERA OFF CACHE BOOL "")
+endif()
```

Comment:
Can this go in the shared CMakeLists.txt (the one included on line 3)?

Reply:
Yes, fixed.